typescript/infra/config/environments/mainnet3/warp/configGetters/getSuperseedWarpConfigs.ts

```diff
@@ -0,0 +1,63 @@
+import { ethers } from 'ethers';
+
+import { ChainMap, HypTokenRouterConfig, TokenType } from '@hyperlane-xyz/sdk';
+import { Address } from '@hyperlane-xyz/utils';
+
+import {
+  RouterConfigWithoutOwner,
+  tokens,
+} from '../../../../../src/config/warp.js';
+
+const safeOwners: ChainMap<Address> = {
+  ethereum: '0x11BEBBf509248735203BAAAe90c1a27EEE70D567',
+  superseed: '0x6652010BaCE855DF870D427daA6141c313994929',
+  optimism: '0x0D493D7E51212bbBF0F1ca4bcfA1E5514C7fEF10',
+};
+
+export const getEthereumSuperseedUSDTConfig = async (
+  routerConfig: ChainMap<RouterConfigWithoutOwner>,
+): Promise<ChainMap<HypTokenRouterConfig>> => {
+  const ethereum: HypTokenRouterConfig = {
```

Comment:
Probably minor point, but I'd prefer a just a single object in the return without naming the objects, reads easier